.circleci/config.yml

```diff
@@ -270,8 +271,19 @@ jobs:
               . activate habitat; cd habitat-lab
               export PYTHONPATH=.:$PYTHONPATH
               export MULTI_PROC_OFFSET=0 && export MAGNUM_LOG=quiet && export HABITAT_SIM_LOG=quiet
-              python -m pytest --cov-report=xml --cov-report term  --cov=./
+              pytest test/ --cov-report=xml --cov-report term  --cov=./
       - codecov/upload
+      - run:
+          name: Run HITL tests
+          no_output_timeout: 60m
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat; cd habitat-lab
+              ln -s ../habitat-sim/data data
```

Comment:
Isn't this link already created in api test job? Are you expecting concurrency?

Reply:
You're correct. It should work without the previous step.
I'm not expecting concurrency.